minio/mcs
# Added API to retrieve user information

## What does this do?
Added an API to retrieve user information from minioadmin

## How to test
1. Go into Postman, set verb as `GET` & URL as `api/v1/users/{user_name}`
2. Set Auth Key and send request
3. You should get the user information
